storm-core/src/clj/backtype/storm/cluster.clj

```diff
@@ -518,6 +540,13 @@
           (catch KeeperException e
             (log-warn-error e "Could not teardown errors for " storm-id))))
 
+      (teardown-topology-log-config!
```

Comment:
@Parth-Brahmbhatt. Currently, I am calling, the `teardown-topology-log-config!` is called from within `do-cleanup ` method on `nimbus.clj`.